packages/app/src/runner/events/telemetry.ts

```diff
@@ -0,0 +1,59 @@
+import { telemetry } from '@packages/telemetry/src/browser'
+
+export const addTelemetryListeners = (getCypress) => {
+  const Cypress = getCypress()
+
+  Cypress.on('test:before:run', (attributes, test) => {
+    // we emit the 'test:before:run' events within various driver tests
+    if (test?.fullTitle()) {
```

Comment:
If we are worried about telemetry going sideways, we should probably wrap this in a `try/catch` instead of a check on the existence of the `fullTitle`.

Reply:
We can do that too, but this also guards against malformed data. Don't send a span if you don't have a title.